cron/index.js

```diff
@@ -104,6 +104,11 @@ module.exports = {
         intervalSec:
           config.cronOverrideAllIntervalsSec || config.cronIntervalWorkspaceHostTransitionsSec,
       },
+      {
+        name: 'cleanTimeSeries',
+        module: require('./cleanTimeSeries'),
+        intervalSec: 'daily',
```

Comment:
Can we add a config variable for this interval and made it default to something like 20 minutes? I don't want to lock the table for very long when we have other writers (the load reporting cron jobs, in particular).

Reply:
Done in 5d24a37 (I opted to do it every 10 minutes).